packages/components/src/components/dropdown/dropdown.ts

```diff
@@ -468,9 +473,12 @@ export default class SdDropdown extends SolidElement {
         transform-origin: left;
       }
 
-      /* When users slot a menu, make sure it conforms to the popup's auto-size */
-      ::slotted(sd-menu),
-      :host(:not([no-auto-size])) ::slotted(*) {
+      /*
+      * Make sure that slotted content conforms to the popup's auto-size if:
+      * - it's an sd-menu
```

Comment:
do we have an sd-menu in the future?

Reply:
I created a follow-up ticket to either remove logic + tests regarding `sd-menu` or build the component: https://github.com/solid-design-system/figma/issues/111